src/nft/components/sell/select/SelectPage.tsx

```diff
@@ -1,3 +1,380 @@
+import clsx from 'clsx'
+import { AnimatedBox, Box } from 'nft/components/Box'
+import { assetList } from 'nft/components/collection/CollectionNfts.css'
+import { LoadingSparkle } from 'nft/components/common/Loading/LoadingSparkle'
+import { Center, Column, Row } from 'nft/components/Flex'
+import { VerifiedIcon } from 'nft/components/icons'
+import { subhead, subheadSmall } from 'nft/css/common.css'
+import { useBag, useIsMobile, useSellAsset, useSellPageState, useWalletBalance, useWalletCollections } from 'nft/hooks'
+import { fetchWalletAssets } from 'nft/queries'
+import { fetchMultipleCollectionStats, OSCollectionsFetcher } from 'nft/queries'
+import { SellPageStateType, WalletAsset } from 'nft/types'
+import { Dispatch, FormEvent, SetStateAction, useEffect, useMemo, useReducer, useState } from 'react'
+import InfiniteScroll from 'react-infinite-scroll-component'
+import { useInfiniteQuery, useQuery } from 'react-query'
+
+import * as styles from './SelectPage.css'
+
+const formatEth = (price: number) => {
```

Comment:
nit: i believe we have a similar function here: `utils/formatDollarAmount`

Reply:
None of the functions in formatDollarAmount handle the 1000s -> K formatting, editing those functions to support this could cause other unintended behaviours